app/lib/katello/validators/product_content_validator.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module Validators
+    class ProductContentValidator < ActiveModel::Validator
```

Comment:
There may be a better way to do this via the usual validation methods, but couldn't quite get it working. Let me know if you see a better way.

Reply:
I think you can do this via simply: 
```
validates :content_id, :presence => true, :uniqueness => {:scope => :product_id}
```

Reply:
that'd be in the ProductContent model

Reply:
Not quite the same validation. See my tests for the product content model. I think we need both ... and I probably should add one for the Product attribute as well.